docker/bitcoin-regtest/entrypoint.sh

```diff
@@ -19,11 +19,11 @@ cd /opt/coins/blockbook/bitcoin_regtest
 
 echo "Starting blockbook service"
 bin/blockbook \
-    -blockchaincfg=config/blockchaincfg.json \
+    -blockchaincfg=/opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json \
```

Comment:
@vdovhanych do we need the full path actually?

Reply:
i've put it here just to b sure that valid config file will be used, see Dockerfile
```
COPY ./docker/bitcoin-regtest/blockchaincfg.json /opt/coins/blockbook/bitcoin_regtest/config/blockchaincfg.json
```
it was not obvious to me where `config/blockchaincfg.json` is pointing

Reply:
maybe it belongs to previous commit, idk... or i can cange it back

Reply:
i think yes, as it's launching it from the bin and uses config that is somewhere else. I think that is should or hast to be full path.